routes/users.js

```diff
@@ -45,4 +44,69 @@ router.get("/asset/:userid", verifyToken, async function (req, res, next) {
   }
 });
 
+router.post("/order/:userid", verifyToken, async function (req, res, next) {
+  try {
+    const { userid } = req.params;
+    const user = await User.findById(userid).lean().exec();
+
+    const userTransactionHistory = user.transactionHistory;
+    delete req.body.authorization;
+    const newTransactionHistory = req.body;
+    userTransactionHistory.push(newTransactionHistory);
+
+    const userCash = user.asset.cash;
```

Comment:
이 부분도 50번째 줄에서 구조분해할당한 값을 쓸 수 있을 거 같아요.
```suggestion
    const userCash = asset.cash;
```
가 될 거 같네요 !